code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -405,15 +397,15 @@
 	private MasterFacet masterFacet = FacetLibrary.getFacet(MasterFacet.class);
 	private AutoEquipmentListFacet autoListEquipmentFacet = FacetLibrary.getFacet(AutoEquipmentListFacet.class);
 	private FollowerFacet followerFacet = FacetLibrary.getFacet(FollowerFacet.class);
-	
+
```

Comment:
I'm really not a fan of changing spacing.  In the past this has created a lot of thrashing between different committers as different IDEs have different spacing and formatting rules.  We should try to focus on functional changes as much as we can and minimize these types of changes unless something is wildly out of whack.